ambari-metrics/ambari-metrics-timelineservice/conf/unix/ambari-metrics-collector

```diff
@@ -309,6 +313,60 @@ function start()
     fi
   }
 
+function start_upgrade()
+{
+  hadoop_java_setup
+
+	CLASS='org.apache.ambari.metrics.core.timeline.upgrade.MetricsSchemaUpgrade'
```

Comment:
We are not really upgrading schema here right, better to call this timeline.upgrade.core.MetricsDataMigrator ?

Reply:
+1 for the above suggestion